### PR TITLE
[Kotlin Migration] ReviewerUi 

### PR DESCRIPTION
**Pull request by codingtosh**
[Kotlin Migration] ReviewerUi

### CODE REVIEW

**Review by mikehardy — APPROVED** [review]

File: AnkiDroid/src/main/java/com/ichi2/anki/reviewer/ReviewerUi.kt [hunk 1]
```diff
@@ -14,24 +14,26 @@
...
-         * so we show the button get deactivated. */
+         * so we show the button get deactivated.  */
         SLOW
     }
-    ControlBlock getControlBlocked();
-    boolean isControlBlocked();
 
-    boolean isDisplayingAnswer();
+    val controlBlocked: ControlBlock?
+    fun isControlBlocked(): Boolean
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/anki/reviewer/ReviewerUi.kt:37]: cleanup attr: this can probably be converted to a property

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
tiny change

File: AnkiDroid/src/main/java/com/ichi2/anki/reviewer/ReviewerUi.kt (hunk 1 above)
Comment (codingtosh) [reply AnkiDroid/src/main/java/com/ichi2/anki/reviewer/ReviewerUi.kt:40]: done, PTAL.

**Review by david-allison-1 — APPROVED** [review]

**Comment by david-allison-1** [comment]
Thanks!!

**Pull request merged**